tfworker/commands/base.py

```diff
@@ -62,7 +62,17 @@ def __init__(
             self._temp_dir,
             tf_version_major,
         )
-        self._plugins = PluginsCollection(rootc.plugins_odict, self._temp_dir)
+        plugins_odict = OrderedDict()
+        for provider in rootc.providers_odict:
+            raw_version = rootc.providers_odict[provider]['vars']['version']
+            version = raw_version.split(' ')[-1]
+            vals = {'version': version}
+            base_url = rootc.providers_odict[provider].get('baseURL')
+            if base_url:
+                vals['baseURL'] = base_url
+            plugins_odict[str(provider)] = vals
+        print(f'plugins_odict: {plugins_odict}')
```

Comment:
can we `click.secho` here for consistency?

Reply:
That was for debugging so I removed it.